app-backend/batch/src/main/scala/groundwork/CreateTaskGrid.scala

```diff
@@ -0,0 +1,96 @@
+package com.rasterfoundry.batch.groundwork
+
+import com.rasterfoundry.batch.Job
+import com.rasterfoundry.database.util.RFTransactor
+import com.rasterfoundry.database.{
+  AnnotationProjectDao,
+  ProjectDao,
+  TaskDao,
+  UserDao
+}
+import com.rasterfoundry.datamodel.{Task, TaskStatus}
+
+import cats.data.OptionT
+import cats.effect.{IO, LiftIO}
+import cats.implicits._
+import com.typesafe.scalalogging.LazyLogging
+import doobie.implicits._
+import doobie.{ConnectionIO, Transactor}
+
+import java.util.UUID
+
+class CreateTaskGrid(
+    annotationProjectId: UUID,
+    taskSizeMeters: Double,
+    xa: Transactor[IO]
+) extends LazyLogging {
+
+  private def debug(s: String): ConnectionIO[Unit] =
+    LiftIO[ConnectionIO].liftIO(
+      IO { logger.debug(s) }
```

Comment:
Nice, maybe some day we'll switch to `log4cats`...

Reply:
Someday... I thought the adventure of figuring out which log4cats version I needed for our cats version wasn't worth the effort :man_shrugging: